dashboard/test/controllers/api/v1/projects/section_projects_controller_test.rb

```diff
@@ -35,31 +35,35 @@ class Api::V1::Projects::SectionProjectsControllerTest < ActionController::TestC
       then.returns(stub_storage_apps)
   end
 
-  test 'student cannot access section projects' do
-    sign_in(@student)
-    get :index, params: {section_id: @section.id}
-    assert_response :forbidden
-  end
+  test_user_gets_response_for(
```

Comment:
Yes, because each one is a different user. I'm open to suggestions on condensing it if you have any ideas that preserve readability.